libraries/AP_Param/AP_Param.cpp

```diff
@@ -38,7 +38,7 @@ extern const AP_HAL::HAL &hal;
 #define ENABLE_DEBUG 1
```

Comment:
This was discussed this week in the dev call. Many developers adding parameters make mistakes and Tridge was getting a lot of support requests because of it. Turning debug on reduced it since developers more easily see what is wrong.